src/client/index.ts

```diff
@@ -1,8 +1,89 @@
+/**
+ * @module Index
+ */
+
 import {tryServiceDispatch, eventEmitter} from './connection';
 import {APITopic} from './internal';
-import {NotificationOptions, Notification} from './Notification';
-import {NotificationEvent} from './Notification';
 
+import {NotificationOptions, Notification, NotificationClickedEvent, NotificationClosedEvent, NotificationButtonClickedEvent} from './index';
+import {NotificationEvent} from './index';
+
+/**
+ * Configuration options for constructing a Notifications object.
```

Comment:
I feel like we should either fill out this doc comment, or add more details about options to the `create` docs. Is it possible to add comments describing each property?

Reply:
Specifically, we should explain that if an ID is not given we will generate one for you.

Reply:
I have added descriptions for each property now